### PR TITLE
Image component updates

### LINKED ISSUE #463: Use react-native-fast-image for caching and Animated Image component improvements.

### PR DESCRIPTION
**Pull request by kamolesh97**
closes #463

### CODE REVIEW

**Comment by kamolesh97** [comment]
@sangameshsomawar _a please review and merge.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Storybook: https://neetoui-rn.onrender.com/
 ## Installation
 
 ```
-yarn add @bigbinary/neetoui-rn react-native-svg react-native-webview @shopify/flash-list react-native-reanimated
+yarn add @bigbinary/neetoui-rn react-native-svg react-native-webview @shopify/flash-list react-native-reanimated react-native-fast-image
```
Comment (sangameshsomawar) [inline README.md:16]: @kamolesh97 also needs to add in 2 more places in getting started markdown file.
https://github.com/bigbinary/neeto-ui-rn/blob/master/docs/pages/1.getting-started.md

**Review by sangameshsomawar — APPROVED** [review]
@kamolesh97 LGTM, please updated md file and merge.

**Comment by kamolesh97** [comment]
@bigbinary/neeto-rn please make sure to install the peer-dependency ```react-native-fast-image``` whenever using this component.

**Pull request merged**